node/podutils/env.go

```diff
@@ -71,9 +71,8 @@ const (
 
 var masterServices = sets.NewString("kubernetes")
 
-// populateEnvironmentVariables populates the environment of each container (and init container) in the specified pod.
-// TODO Make this the single exported function of a "pkg/environment" package in the future.
-func populateEnvironmentVariables(ctx context.Context, pod *corev1.Pod, rm *manager.ResourceManager, recorder record.EventRecorder) error {
+// PopulateEnvironmentVariables populates the environment of each container (and init container) in the specified pod.
+func PopulateEnvironmentVariables(ctx context.Context, pod *corev1.Pod, rm *manager.ResourceManager, recorder record.EventRecorder) error {
```

Comment:
Oh, maybe one thing here. `resource` is an internal package.

Reply:
I don't see `resource` being referenced?

Reply:
Or are you suggesting we make this an internal package?

Reply:
Sorry, `manager.ResourceManager` (brain fart).

Reply:
Fixed. I think.

Reply:
Change works for me. Would still like to get something like @justnoise's change in that makes this a public interface that providers can choose to use... but this is good for now.

Thanks!